### PR TITLE
feat(bmd): Prompt pollworker to open/close polls on VxScan if no tally report on card

### PR DESCRIPTION
**Pull request by taravancil**
## Overview
This completes the first task listed in #1311 

When a pollworker inserts a card _without_ a tally report, when they try to open/close the polls, we first show a modal instructing them to open/close the polls on VxScan.

## Testing Plan 
Manually tested and added a test.

## Checklist
- [x] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [x] I have added JSDoc comments to any newly introduced exports

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

**Review by benadida — APPROVED** [review]

File: frontends/bmd/src/pages/poll_worker_screen.tsx [hunk 1]
```diff
@@ -550,6 +561,38 @@ export function PollWorkerScreen({
...
+                >
+                  Cancel
+                </Button>
+                <Button
+                  onPress={() => {
+                    togglePollsOpen();
+                    setIsShowingVxScanPollsOpenModal(false);
+                  }}
+                >
+                  {isPollsOpen ? 'Close' : 'Open'} Polls Now
```
Comment (eventualbuddha) [inline frontends/bmd/src/pages/poll_worker_screen.tsx:590]: Same here:
```suggestion
                  {isPollsOpen ? 'Close Polls Now' : 'Open Polls Now'}
```

**Review by eventualbuddha — COMMENTED** [review]
Some code-level feedback. I think the modal language is a little confusing, personally, but I'll leave that to @benadida and @mattroe.

File: frontends/bmd/src/pages/poll_worker_screen.tsx [hunk 2]
```diff
@@ -550,6 +561,38 @@ export function PollWorkerScreen({
...
+        {isShowingVxScanPollsOpenModal && (
+          <Modal
+            centerContent
+            content={
+              <Prose textCenter id="modalaudiofocus">
+                <h1>{isPollsOpen ? 'Close' : 'Open'} Polls on VxScan</h1>
+                <p>
+                  {isPollsOpen ? 'Close' : 'Open'} polls on VxScan first before{' '}
+                  {isPollsOpen ? 'closing' : 'opening'} VxMark.
+                </p>
```
[the remark this answers is not in the document]
Comment (taravancil) [reply frontends/bmd/src/pages/poll_worker_screen.tsx:573]: Good call!

File: frontends/bmd/src/pages/poll_worker_screen.test.tsx [hunk 3]
```diff
@@ -1267,3 +1267,64 @@ test('navigates to System Diagnostics screen', () => {
...
+  });
+
+  fireEvent.click(screen.getByText('Open Polls for Center Springfield'));
+
+  // Should show the modal and not open/close polls
+  expect(togglePollsOpen).not.toHaveBeenCalled();
+  screen.getByText('Open Polls on VxScan');
+
+  // Clicking Cancel closes the modal
+  fireEvent.click(screen.getByText('Open Polls for Center Springfield'));
```
[the remark this answers is not in the document]
Comment (taravancil) [reply frontends/bmd/src/pages/poll_worker_screen.test.tsx]: Whoops that's extraneous, removed.

**Review by mattroe — COMMENTED** [review]
Suggested alternative copy after seeing the screen:
* Header (what is the state/why is this screen being shown to the user): "No Polls [Opened/Closed] Report on Card"
* Text (what is the action the user should do): "[Open/Close] polls on VxScan to save tally report before [opening/closing] polls on VxMark."
* Button 1 (clarifying this is for VxMark only): "[Open/Close] VxMark Now"

**Comment by adghayes** [comment]
@mattroe I suggest that instead of "No Tally Report on Card" we use "No Polls Opened Report on Card" and "No Polls Closed Report on Card" so that the meaning might be clearer to the poll worker, as poll workers probably don't know what "Tally Report" means.

**Comment by mattroe** [comment]
@adghayes Good call. We should the terminology that's already on the physical reports. Editing previous comment to reflect this.

**Comment by adghayes** [comment]
@mattroe I made the assumption that we want to edit both the header and the "action" text as in these screenshots (so it doesn't say "tally report" anywhere). 

Screen Shot 2022-07-18 at 6 30 00 PM
Screen Shot 2022-07-18 at 6 29 21 PM

**Pull request merged**